minecraft_servers/craftergang/manifest.json

```diff
@@ -14,13 +14,20 @@
     "teamspeak": "craftergang.de",
     "tiktok": "craftergang",
     "youtube": "https://www.youtube.com/@craftergang",
+    "web_support": "https://forum-craftergang.de",

```

Comment:
The support page cannot be a forum; provide a faq, contact-form or similar, or remove

Reply:
I Have changed